New version: REINERSCT.cyberJackDriverPackage version 1.4.0

nimra98 opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue? #186061

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/186060)

nimra98 committed:
New version: REINERSCT.cyberJackDriverPackage version 1.4.0

nimra98 commented:
@microsoft-github-policy-service agree

nimra98 committed:
Update REINERSCT.cyberJackDriverPackage.locale.en-US.yaml

Change Release Notes

stephengillie commented:
@wingetbot run

nimra98 committed:
Update REINERSCT.cyberJackDriverPackage.installer.yaml

remove interactive install, add eleveationRequirement and UpgradeBehavior

stephengillie commented:
Automatic Validation ended with:


(Automated response - build 897.)

stephengillie reviewed (changes requested):
> Automatic Validation ended with: Edit: ![image](https://private-user-images.githubusercontent.com/114952565/381808038-dd48667b-fee0-4f0f-9831-581deef42165.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzNDQxMzcsIm5iZiI6MTczMDM0MzgzNywicGF0aCI6Ii8xMTQ5NTI1NjUvMzgxODA4MDM4LWRkNDg2NjdiLWZlZTAtNGYwZi05ODMxLTU4MWRlZWY0MjE2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAzMVQwMzAzNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTk5ZWRiMjJhYTNjOGY2ZDJkYTZiNzE0ZWJlZjg2YWVhN2M5MjZiZWU1ZTNjM2M2MGE3OGEyMTI5ODFkYjY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rxjRSTo_EdsUpxnQkA1yG2qiBVIRtFLij4SrrM2QCZc)

Edit: 
Hi @nimra98,

Our pipeline uses 32-bit OSes to validate `x86` packages. 

I meant to suggest a change, but forgot to click "Start a review".

stephengillie commented on manifests/r/REINERSCT/cyberJackDriverPackage/1.4.0/REINERSCT.cyberJackDriverPackage.installer.yaml:
```diff
@@ -0,0 +1,26 @@
+ElevationRequirement: elevationRequired
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://dp.reiner-sct.com/downloads/dp/dp_1_4_0.exe
+  InstallerSha256: 0F27AB139971A275EC9E8AC94D1D35C48F545854C4D0DA78D439CBB69A6558FB
```
```suggestion
```

nimra98 committed:
Update manifests/r/REINERSCT/cyberJackDriverPackage/1.4.0/REINERSCT.cyberJackDriverPackage.installer.yaml

Remove x86 support

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

nimra98 committed:
Update REINERSCT.cyberJackDriverPackage.installer.yaml

Add arm64 support on Win11

nimra98 commented:
> > Automatic Validation ended with: Edit: ![image](https://private-user-images.githubusercontent.com/114952565/381808038-dd48667b-fee0-4f0f-9831-581deef42165.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAzNDQxMzcsIm5iZiI6MTczMDM0MzgzNywicGF0aCI6Ii8xMTQ5NTI1NjUvMzgxODA4MDM4LWRkNDg2NjdiLWZlZTAtNGYwZi05ODMxLTU4MWRlZWY0MjE2NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAzMVQwMzAzNTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MTk5ZWRiMjJhYTNjOGY2ZDJkYTZiNzE0ZWJlZjg2YWVhN2M5MjZiZWU1ZTNjM2M2MGE3OGEyMTI5ODFkYjY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rxjRSTo_EdsUpxnQkA1yG2qiBVIRtFLij4SrrM2QCZc)
> 
> Edit: Hi @nimra98,
> 
> Our pipeline uses 32-bit OSes to validate `x86` packages.
> 
> I meant to suggest a change, but forgot to click "Start a review".

Thanks for the help. I missed that the installer now only supports x64 Win 10 and x64/arm64 Win11, as I only copied the old entry and modified the installer location.
I also added the arm64 for Win11 and hope the pipeline runs through.
